src/rollup/display-conditions/meta-checker.js

```diff
@@ -0,0 +1,46 @@
+/** @module pathfora/display-conditions/meta-checker */
+
+// utils
+import document from '../dom/document';
+
+/**
+ * Check if the current page contains the meta
+ * tag and value provided
+ *
+ * @exports metaChecker
+ * @params {array} phrases
+ * @returns {boolean}
+ */
+export default function metaChecker (phrases) {
+  var meta = document.querySelectorAll('meta');
+
+  for (var j = 0; j < phrases.length; j++) {
+    var rule = phrases[j],
+        phraseValid = false;
+
+    for (var i = 0; i < meta.length; i++) {
+      for (var key in rule) {
+        if (rule.hasOwnProperty(key)) {
+          var val = meta[i].getAttribute(key);
+
+          if (!val || val !== rule[key]) {
+            phraseValid = false;
+            break;
+          } else {
+            phraseValid = true;
+          }
+        }
```

Comment:
just a hunch, have not tested. but we probablyi need a `phraseValid = false` here and a continue after the `phraseValid = true` on line 30 to ensure its not being shown when one of two conditions is met....i think